docs/proposals/resource-catalog.md

```diff
@@ -0,0 +1,222 @@
+
+
+We can reshape the catalog resource to more easily express them, secure them, and claim them.
+
+# Goals
+ 1.  cluster-admin should control which catalog exist.
+ 2.  cluster-admin should be able/disable per-catalog exposure on a per-project granularity.
```

Comment:
Is this use case saying that there should be a way to disallow resources from project X from being included in catalog Y?